lib/oli_web/templates/editor_test/index.html.eex

```diff
@@ -0,0 +1,7 @@
+<script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/components.js") %>"></script>
+
+<div id="app"/>
```

Comment:
We might want to call this id something less generic than app, like `<div id="testEditor"/>` since there could be other react component mounts in the rendered html in the future

Reply:
Agreed, done.